android/src/test/java/com/segment/analytics/kotlin/android/EventsFileTests.kt

```diff
@@ -32,6 +33,8 @@ class EventsFileTests {
 
     init {
         mockkStatic(Instant::class)
+        mockkStatic(::dateTimeNowString)
+        every { dateTimeNowString() } returns Date(0).toInstant().toString()
```

Comment:
do we need to replace `toInstant` too? looks like it's returning an `Instant` value. would that require desugaring?

Reply:
Not in our tests. I let all of those keep the use of the Instant classes. Instant was only removed from code that runs on the device.

Reply:
Yes. It is still available because the test run locally using java 11. I have updated the code to remove desugaring from the android build.gradle and re-run the test and they still all pass.